cmd/version/version.go

```diff
@@ -16,7 +16,11 @@ func GetVersionCmd() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			v := cautils.NewIVersionCheckHandler()
 			v.CheckLatestVersion(cautils.NewVersionCheckRequest(cautils.BuildNumber, "", "", "version"))
-			fmt.Fprintln(os.Stdout, "Your current version is: "+cautils.BuildNumber)
+			fmt.Fprintf(os.Stdout,
+				"Your current version is: %s [git enabled in build: %t]\n",
```

Comment:
I think we should print if git is enabled or not only in debug mode

Reply:
ok good idea. Doing this

Reply:
kubescape --logger debug version
[warning] unknown build number, this might affect your scan results. Please make sure you are updated to latest version
Your current version is:  
[debug] [git disabled in build]